relay-server/src/extractors/request_meta.rs

```diff
@@ -280,22 +288,7 @@ impl RequestMeta {
             user_agent: Some(crate::constants::SERVER.to_owned()),
             no_cache: false,
             start_time: Instant::now(),
-        }
-    }
-
-    #[cfg(test)]
-    // TODO: Remove Dsn here?
-    pub fn new(dsn: relay_common::Dsn) -> Self {
```

Comment:
moved to mod tests{}

Reply:
In general, I'd suggest to keep PRs focused on their scope. More small and focused PRs is better than a few big PRs.